Gemfile.lock

```diff
@@ -57,23 +58,26 @@ GEM
       thor (~> 0.19)
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
-    public_suffix (5.0.3)
+    public_suffix (5.0.4)
     rb-fsevent (0.11.2)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
     rexml (3.2.6)
-    rouge (4.1.3)
+    rouge (4.2.0)
     safe_yaml (1.0.5)
     sass (3.4.25)
-    sass-embedded (1.68.0-x86_64-darwin)
+    sass-embedded (1.69.5-arm64-darwin)
+      google-protobuf (~> 3.23)
+    sass-embedded (1.69.5-x86_64-darwin)
       google-protobuf (~> 3.23)
     terminal-table (3.0.2)
       unicode-display_width (>= 1.1.1, < 3)
     thor (0.20.3)
-    unicode-display_width (2.4.2)
+    unicode-display_width (2.5.0)
     webrick (1.8.1)
 
 PLATFORMS
+  arm64-darwin-23
   x86_64-darwin-21
```

Comment:
@ryanwoldatwork @XavierMetichecchia thanks for the considerations. I didn't realize the previous platform was removed via `bundle install`. 

I've reverted that change in 8e5536c and have added my local platform in 52d0489 (_via the command `bundle lock --add-platform arm64-darwin-23`_). This should improve compatibility moving forward.